src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1092,6 +1092,29 @@ protected static ImmutableArray<string> CheckTupleElementNames(int cardinality,
 
         #endregion
 
+        #region Operations
+
+        /// <summary>
+        /// Gets the low-level operation corresponding to the method's body.
+        /// </summary>
+        /// <param name="method">The method symbol.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>The low-level operation corresponding to the method's body.</returns>
+        internal IOperation GetOperation(IMethodSymbol method, CancellationToken cancellationToken = default)
```

Comment:
>GetOperation [](start = 28, length = 12)

We should test this API for all different kinds of methods that can be declared in source: constructors, destructors, accessors, partial methods, etc. Also should test it for methods that have their bodies synthesized by compilers: default constructors, auto-property accessors, etc. For both languages.